### PR TITLE
chore: cleaning up unused and redundant sampels and their ITs

### PR DESCRIPTION
**Pull request by munkhuushmgl**
go/legacy-automl-sample-testing

Devsite updates:
cl/355497151
cl/355469526

### CODE REVIEW

File: samples/snippets/src/main/java/com/google/cloud/translate/automl/DatasetApi.java [hunk 1]
```diff
@@ -43,156 +39,6 @@
  */
 public class DatasetApi {
 
-  // [START automl_translate_create_dataset]
```
Comment (telpirion) [inline samples/snippets/src/main/java/com/google/cloud/translate/automl/DatasetApi.java:46]: We need to keep at least one copy of automl_translate_create_dataset around.

File: samples/snippets/src/main/java/com/google/cloud/translate/automl/ModelApi.java [hunk 2]
```diff
@@ -47,48 +41,6 @@
  */
 public class ModelApi {
 
-  // [START automl_translate_create_model]
```
Comment (telpirion) [inline samples/snippets/src/main/java/com/google/cloud/translate/automl/ModelApi.java:50]: We need to keep one copy of automl_translate_create_model around.

**Review by telpirion — COMMENTED** [review]
Please verify that we have another version of automl_translate_create_dataset and automl_create_model somewhere in this repo.

File: samples/snippets/src/main/java/com/google/cloud/translate/automl/DatasetApi.java (hunk 1 above)
Comment (munkhuushmgl) [reply samples/snippets/src/main/java/com/google/cloud/translate/automl/DatasetApi.java:46]: https://github.com/googleapis/java-automl/blob/fb9dad14aac1d684ea136ccca158798545ea26f3/samples/snippets/src/main/java/beta/automl/TablesCreateDataset.java#L19

File: samples/snippets/src/main/java/com/google/cloud/translate/automl/ModelApi.java (hunk 2 above)
Comment (munkhuushmgl) [reply samples/snippets/src/main/java/com/google/cloud/translate/automl/ModelApi.java:50]: https://github.com/googleapis/java-automl/blob/fb9dad14aac1d684ea136ccca158798545ea26f3/samples/snippets/src/main/java/beta/automl/TablesCreateModel.java#L19

**Review by telpirion — APPROVED** [review]
Sorry for the confusion. LGTM!

**Review by lesv — APPROVED** [review]

**Pull request merged**